definitions/npm/effector-react_v22.x.x/flow_v0.83.x-/effector-react_v22.x.x.js

```diff
@@ -0,0 +1,119 @@
+declare module 'effector-react' {
+  //@flow strict
```

Comment:
don't need this at all in the file. But also flow pragmas should only be referenced at the top of the file. I imagine if you removed this everything should still work